cmd/root.go

```diff
@@ -220,3 +222,45 @@ func prepConfig() error {
 	}
 	return nil
 }
+
+func exportTransformers() []transformer.StorageTransformerInitializer {
```

Comment:
Maybe it would make sense to name this `exportStorageTransformers`?

Reply:
I think I'll keep the same name, and then have it export all of the initializers. Thinking we can reuse this code in the execute command as well